tmk_core/rules.mk

```diff
@@ -100,10 +100,10 @@ endif
 #CFLAGS += -Wundef
 #CFLAGS += -Wunreachable-code
 #CFLAGS += -Wsign-compare
+CFLAGS += -fstrict-aliasing -ftree-vrp
```

Comment:
I think you could actually add this to your userspace's rules.mk.   
It may be worth going so, so we can get this merged.  (I'm hesitant to merge code into core files, but no issue if this is 100% keymaps and userspace code)

Reply:
@doogle999 Did you want to make this change? 

If so, then we could merge it.

Reply:
```suggestion
```